appscale/tools/parse_args.py

```diff
@@ -339,9 +339,7 @@ def add_allowed_flags(self, function):
         help="the keypair name to use")
       self.parser.add_argument('--test', action='store_true',
         default=False,
-        help="uses a default username and password for cloud admin")
-      self.parser.add_argument('--email',
-        help="the e-mail address to use as the app's admin")
+        help="avoids prompting for user input")
```

Comment:
If true, the prompt is skipped when version is defined: https://github.com/AppScale/appscale-tools/blob/8cf26e68ede22485bd7978a7cac888ec99c70f5c/appscale/tools/appscale_tools.py#L912